definitions.yaml

```diff
@@ -237,6 +237,114 @@ definitions:
         type: object
         additionalProperties: true
 
+  V4GetClusterStatusResponse:
+    type: object
+    description: Object about a cluster's current state
+    properties:
+      aws:
+        type: object
+        description: |
+          AWS specific status information. Only given in AWS installations.
+        properties:
+          availabilityZones:
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: Name of the randomly selected availability zone
+                subnet:
+                  type: object
+                  properties:
+                    private:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Private subnet CIDR allocated for this availability
+                            zone
+                    public:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Public subnet CIDR allocated for this availability
+                            zone
+      cluster:
+        type: object
+        properties:
+          conditions:
+            type: array
+            items:
+              type: object
+              description: TODO
+              properties:
+                lastHeartbeatTime:
+                  type: string
+                  description: Last time the condition was checked and verified
+                lastTransitionTime:
+                  type: string
+                  description: |
+                    Last time the condition transitioned from one state to
+                    another
+                status:
+                  type: string
+                  description: |
+                    Status of the condition. One of True, False, or Unknown.
```

Comment:
Maybe also add what status of True, False or Unknown means.

Reply:
Feel free to suggest a concrete edit here that pleases your personal taste. I do not know what may be right or wrong here.